Automattic/jetpack
# Sharing block: avoid error when a service is not available

## Proposed changes:

This is a check that likely will not be that useful to site owners, since we will be adding services, not necessarily removing them.
However, during development and switching between branches, it can happen that a service you've added to the sharing block is not available anymore.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Apperance > Site Editor > Templates > Single Posts
* Add a sharing block, and some sharing buttons inside that block.
* Save your changes
* Notice your changes on the frontend; all the buttons should be there.
